orchestra/static/orchestra/team-info/team-info-card.html

```diff
@@ -59,6 +68,40 @@ <h3>
                 </button>
               </td>
             </tr>
+            <tr ng-if="teamInfoCard.isProjectAdmin">
+              <td colspan="5">
+                <a
+                  class="btn"
+                  ng-click="teamInfoCard.toggleShowUnassigned()">
+                  {{teamInfoCard.showUnassigned ? 'Hide unassigned' : 'Show unassigned' }}
+                </a>
+              </td>
+            </tr>
+            <tr
+              ng-if="teamInfoCard.showUnassigned"
+              ng-repeat="unassigned in teamInfoCard.unassigned">
```

Comment:
curiously, are these in the same order as they appear in the project management view? i think that's what the team would prefer

Reply:
Now they appear in the same order.

<img width="876" alt="Screen Shot 2020-10-21 at 12 41 18" src="https://user-images.githubusercontent.com/3319967/96682571-d25af600-139a-11eb-9042-2963534f0d66.png">

<img width="1295" alt="Screen Shot 2020-10-21 at 12 41 54" src="https://user-images.githubusercontent.com/3319967/96682585-d7b84080-139a-11eb-9e1a-145651f07012.png">